### ISSUE TITLE
Error on mfrac with linethickness=0

### ISSUE THREAD

**Issue by kwkbtr** [mathjax/MathJax-a11y#161]
Test case: http://jsbin.com/gavehinemu/edit?html,css,output

When the semantic enrichment is enabled, the following markup results in "Math Processing Error":

```xml
<math>
    <mfrac linethickness="0">
        <mo>a</mo>
        <mrow><mi>b</mi></mrow>
    </mfrac>
</math>
```

**Comment by pkra** [comment]
Thanks for the report! `MathJax.Hub.lastError` indicates an upstream issue in speech-rule-engine.

@zorkow can you take a look?

**Comment by pkra** [comment]
Might be related to https://github.com/zorkow/speech-rule-engine/issues/108

**Comment by zorkow** [comment]
It is actually related to this issue:
https://github.com/zorkow/speech-rule-engine/issues/21
and the branch
https://github.com/zorkow/speech-rule-engine/tree/robust_walking
fixes for a number of problems related to tables.

**Comment by pkra** [comment]
Thanks, Volker. Marked `upstream`.

**Comment by pkra** [comment]
@zorkow, I think this is fixed now that we're on SRE v2, correct?

**Comment by zorkow** [comment]
It is.

**Comment by pkra** [comment]
Closed via #183
